druid/src/widget/mod.rs

```diff
@@ -70,3 +70,10 @@ pub use widget_ext::WidgetExt;
 
 mod list;
 pub use crate::widget::list::{List, ListIter};
+
+mod svg;
+#[cfg(feature = "svg")]
+pub use crate::widget::svg::{svg_to_piet, svg_tree_from_path, svg_tree_from_str, Svg};
+
+#[cfg(not(feature = "svg"))]
+pub use crate::widget::svg::FakeSvg as Svg;
```

Comment:
I'd rather see the module feature-guarded in totality rather than playing error-prone and surprising tricks like this, especially since this doesn't even cover the full API.

Reply:
the discussion was that sometimes that leads to confusing errors, but I do agree that it's not a great feeling to have all this extra code for not a lot of tangible benefit.

Reply:
I think having the API silently do nothing (in the very likely event that you're not seeing `log` messages), or change shape in arbitrary ways (the current state, where not all items are stubbed) is strictly worse than having things missing, particularly given that rustdoc can be manipulated to prominently display feature guards.

Reply:
cool, didn't know about that you could show guards in rustdoc.

I'm happy to have this removed, it does make things clearer. Maybe another way to salve this wound would be to turn the `svg` feature on by default?

Reply:
See [tokio](https://docs.rs/tokio/0.2.2/tokio/) for an example of this. Note that it requires nightly rustdoc.

I think enabling it by default would somewhat defeat the point of keeping druid slim.

Reply:
Agree, experience in the Rust ecosystem is that it's *much* harder to turn off default features than it is to opt in.